SpatialGDK/Source/SpatialGDK/Public/Interop/OwnershipCompletenessHandler.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "Misc/Optional.h"
+
+#include "SpatialCommonTypes.h"
+#include "SpatialView/SubView.h"
+
+class USpatialNetDriver;
+
+namespace SpatialGDK
+{
+class ViewCoordinator;
+
+class FOwnershipCompletenessHandler
+{
+public:
+	FOwnershipCompletenessHandler(bool bInIsServer)
```

Comment:
explicit

Reply:
Boolean params like this can be confusing to read at the callsites... another option would be to provide  static factory functions for client and server types.

Reply:
Factory functions do look great in this case, thank you for suggesting this!